test/SlowTests/Client/Subscriptions/ConcurrentSubscriptionsTests.cs

```diff
@@ -59,24 +77,45 @@ public async Task ConcurrentSubscriptions()
                     var con1Docs = new List<string>();
                     var con2Docs = new List<string>();
 
-                    var t = subscription.Run(x =>
+                    var t1 = s1.Run(x =>
                     {
                         foreach (var item in x.Items)
                         {
                             con1Docs.Add(item.Id);
                         }
                     });
-
-                    var _ = secondSubscription.Run(x =>
+                    var t2 = s2.Run(x =>
+                    {
+                        foreach (var item in x.Items)
+                        {
+                            con2Docs.Add(item.Id);
+                        }
+                    });
+                    var t3 = s3.Run(x =>
+                    {
+                        foreach (var item in x.Items)
+                        {
+                            con1Docs.Add(item.Id);
+                        }
+                    });
+                    var t4 = s4.Run(x =>
+                    {
+                        foreach (var item in x.Items)
+                        {
+                            con2Docs.Add(item.Id);
+                        }
+                    });
+                    var t5 = s5.Run(x =>
                     {
                         foreach (var item in x.Items)
                         {
                             con2Docs.Add(item.Id);
                         }
                     });
 
-                    await AssertWaitForTrueAsync(() => Task.FromResult(con1Docs.Count + con2Docs.Count == 6), 6000);
-                    await AssertNoLeftovers(store, id);
+                    WaitForUserToContinueTheTest(store);
+                    // await AssertWaitForTrueAsync(() => Task.FromResult(con1Docs.Count + con2Docs.Count == 6), 6000);
```

Comment:
?

Reply:
I accidentally committed this file
now I pushed w/ force the commit w/o this file